proxy/logging/LogAccessHttp.cc

```diff
@@ -71,7 +71,7 @@ LogAccessHttp::LogAccessHttp(HttpSM *sm)
     m_proxy_resp_content_type_str(nullptr),
     m_proxy_resp_content_type_len(0),
     m_cache_lookup_url_canon_str(nullptr),
-    m_cache_lookup_url_canon_len(0)
+    m_cache_lookup_url_canon_len(-1)
```

Comment:
Maybe, not sure which (if any) should stay zero though. But it seems the URL ones ought to be consistently -1. Let me check that.